distribution/docs/src/main/resources/_contents/_endpoints/csw-endpoint-contents.adoc

```diff
@@ -0,0 +1,2117 @@
+==== CSW Endpoint
+
+The CSW endpoint enables a client to search collections of descriptive information (metadata) about geospatial data and services.
+
+===== Installing CSW Endpoint
+
+The CSW Endpoint is installed by default with a standard installation in the ${ddf-spatial} application.
+
+===== Configuring CSW Endpoint
+
+The CSW endpoint has no configurable properties.
+It can only be installed or uninstalled.
+
+===== CSW Endpoint URL
+
+The CSW endpoint is accessible from ${secure_url}/services/csw.
+
+====== CSW Endpoint Operations
+
+`GetCapabilities` Operation:: The `GetCapabilites` operation is meant to describe the operations the catalog supports and the URLs used to access those operations. The CSW endpoint supports both `HTTP GET` and `HTTP POST` requests for the `GetCapabilities` operation. The response to either request will always be a `csw:Capabilities` XML document. This XML document is defined by the http://schemas.opengis.net/csw/2.0.2/CSW-discovery.xsd[CSW-Discovery XML Schema].
+
+.`GetCapabilities` HTTP GET
+The `HTTP GET` form of `GetCapabilities` uses query parameters via the following URL:
+
+.`GetCapabilities` KVP Encoding
+----
+${secure_url}/services/csw?service=CSW&version=2.0.2&request=GetCapabilities
+----
+
+.`GetCapabilities` HTTP POST
+The `HTTP POST` form of `GetCapabilities` operates on the root CSW endpoint URL (\${secure_url}/services/csw) with an XML message body that is defined by the `GetCapabilities` element of the http://schemas.opengis.net/csw/2.0.2/CSW-discovery.xsd[CSW-Discovery XML Schema].
+
+.`GetCapabilities` XML Request
+[source,xml,linenums]
+----
+<?xml version="1.0" ?>
+<csw:GetCapabilities
+  xmlns:csw="http://www.opengis.net/cat/csw/2.0.2"
+  service="CSW"
+  version="2.0.2" >
+</csw:GetCapabilities>
+----
+
+.`GetCapabilities` Sample Response (`application/xml`)
+[source,xml,linenums]
+----
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
I'm not sure how useful this sample XML actually is since we are already referencing the CSW Discovery XML Schema above. Also, it's really large and doesn't read very well in PDF format.

If we really want to keep it, should be move it to an appendix and refer to it from here?